examples/making-an-eval-command.md

```diff
@@ -109,7 +109,7 @@ client.on("messageCreate", async (message) => {
       message.channel.send(`\`\`\`js\n${cleaned}\n\`\`\``);
     } catch (err) {
       // Reply in the channel with our error
-      message.channel.send(`\`ERROR\` \`\`\`xl\n${cleaned}\n\`\`\``);
+      message.channel.send(`\`ERROR\` \`\`\`xl\n${err.stack}\n\`\`\``);
```

Comment:
1. This doesn't send the actual error.
2. err.stack is `string[]?` meaning it *could* be undefined.

Reply:
> 1. This doesn't send the actual error.
> 2. err.stack is `string[]?` meaning it _could_ be undefined.

1. `Error#stack` does send the actual error message, plus the error stack.
2. Technically `err` is of type `unknown`, so an instance check for `Error` would be appropriate here, but the chances of a non-error type being thrown is unlikely.

Reply:
@RealShadowNova... oops; implementation issue with the way my stack system works. soz!